### DISCUSSION TITLE
Thunderbird

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#12] [Requests] [answered]
Thunderbird website: https://www.thunderbird.net

Thunderbird privacy policy: https://www.mozilla.org/privacy/thunderbird

Logo: https://upload.wikimedia.org/wikipedia/commons/e/e1/Thunderbird_Logo%2C_2018.svg

Description: "Mozilla Thunderbird is a free and open-source cross-platform email client, personal information manager, news client, RSS and chat client developed by the Mozilla Foundation. The project strategy was originally modeled after that of the Mozilla Firefox web browser.

On December 7, 2004, version 1.0 was released, and received more than 500,000 downloads in its first three days of release, and 1,000,000 in ten days.

On July 6, 2012, Mozilla announced the company was dropping the priority of Thunderbird development because the continuous effort to extend Thunderbird's feature set was mostly fruitless. The new development model shifted to Mozilla offering only "Extended Support Releases", which deliver security and maintenance updates, while allowing the community to take over the development of new features.

On December 1, 2015, Mozilla Executive Chair Mitchell Baker announced in a company-wide memo that Thunderbird development needed to be uncoupled from Firefox. She referred to Thunderbird developers spending large efforts responding to changes to Mozilla technologies, while Firefox was paying a tax to support Thunderbird development. She also said that she does not believe Thunderbird has the potential for "industry-wide impact" that Firefox does. At the same time, it was announced that Mozilla Foundation would provide at least a temporary legal and financial home for the Thunderbird project.

Mozilla brought Thunderbird back in-house in an announcement on May 9, 2017, and continued to support its development. The Thunderbird development team expanded by adding several new members and overhauled security and the user interface.

On January 28, 2020, the Mozilla Foundation announced that the project would henceforth be operating from a new wholly owned subsidiary, MZLA Technologies Corporation, in order to explore offering products and services that were not previously possible and to collect revenue through partnerships and non-charitable donations."

**Comment by mayrinck** [comment]
Awesome suggestion. It's my prime choice over the Windows 10 email app. I will add it as soon as possible. 
Thank you for submitting.

**Reply by mayrinck** [reply]
Well, considering the fact that browsers should follow all those rules and things, I guess the same applies to email clients too... In this case, Thunderbird might be a problem to upload to the store...

**Reply by JuliaAgostino** [reply]
> Well, considering the fact that browsers should follow all those rules and things, I guess the same applies to email clients too... In this case, Thunderbird might be a problem to upload to the store...

Why would the browser rules apply to email clients?

**Reply by mayrinck** [reply]
because usually they use browser engines to render the emails, so the rules may apply...

**Reply by JuliaAgostino** [reply]
> because usually they use browser engines to render the emails, so the rules may apply...

It's at least worth trying though, right?

**Reply by JuliaAgostino** [reply]
> Well, considering the fact that browsers should follow all those rules and things, I guess the same applies to email clients too... In this case, Thunderbird might be a problem to upload to the store...

Could we reconsider this one now that Mozilla Firefox is officially listed in the Microsoft Store?

**Comment by JuliaAgostino** [comment, marked as answer]
Well, it looks like Mozilla plans on putting Thunderbird in the Microsoft Store anyway: https://twitter.com/mozthunderbird/status/1527062053910941696?s=20
